### DISCUSSION TITLE
Whether to support rotating frame target detection?

### DISCUSSION THREAD

**Discussion by ZSBSB** [obss/sahi#1090] [Q&A] [answered]
Thank you very much for the work you have done, does SAHI currently support target detection for yolov11 rotating frames?

**Comment by fcakyon** [comment, marked as answer]
Yes @ZSBSB , Ultralytics YOLOv5U/YOLOv8/YOLO11 OBB models are supported with latest sahi release: https://github.com/obss/sahi/releases/tag/0.11.20

Usage is similar to Ultralytics det/seg models: https://github.com/obss/sahi/blob/main/demo/inference_for_ultralytics.ipynb

**Comment by ZSBSB** [comment]
Thank you for your reply. Through your guidance, I have found a solution.
&nbsp;


shaobo_zhang
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "fatih ***@***.***&gt;; 
发送时间: 2024年12月17日(星期二) 凌晨2:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [obss/sahi] Whether to support rotating frame target detection? (Discussion #1090)















***@***.***，超解液 YOLOv5U/YOLOv8/YOLO11 OBB 型号支持最新的 sahi 版本:https://github.com/obss/sahi/releases/tag/0.11.20

用途与超解液 det/seg 型号相似:https://github.com/obss/sahi/blob/main/demo/inference_for_ultralytics.ipynb

-
直接回复此邮件，在GitHub上查看它，或取消订阅.
***@***.*** &gt;
